x/ibc/light-clients/07-tendermint/types/genesis.go

```diff
@@ -0,0 +1,21 @@
+package types
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	clienttypes "github.com/cosmos/cosmos-sdk/x/ibc/core/02-client/types"
+	"github.com/cosmos/cosmos-sdk/x/ibc/core/exported"
+)
+
+// ExportMetadata exports all the processed times in the client store so they can be included in clients genesis
+// and imported by a ClientKeeper
+func (cs ClientState) ExportMetadata(store sdk.KVStore) []exported.GenesisMetadata {
+	gm := make([]exported.GenesisMetadata, 0)
+	IterateProcessedTime(store, func(key, val []byte) bool {
+		gm = append(gm, clienttypes.NewGenesisMetadata(key, val))
```

Comment:
Does this need to be sorted? cc @fedekunze

Reply:
it will already be sorted by key